web/src/beta/features/Editor/tabs/widgets/SidePanel/ContainerSettings/hooks.ts

```diff
@@ -0,0 +1,19 @@
+import { useCallback } from "react";
+
+import { useWidgetsFetcher } from "@reearth/services/api";
+import type { WidgetAreaState } from "@reearth/services/state";
+
+export default ({ sceneId }: { sceneId: string }) => {
```

Comment:
Can you move the selectedWidgetArea state to here? There is no need for it in the Sidepanel 🙏🏻